test/Microsoft.TestPlatform.Common.UnitTests/ExtensionFramework/TestPluginDiscovererTests.cs

```diff
@@ -125,6 +127,21 @@ public void GetTestExtensionsInformationShouldNotAbortOnFaultyExtensions()
             Assert.That.DoesNotThrow(() =>this.testPluginDiscoverer.GetTestExtensionsInformation<FaultyTestExecutorPluginInformation, ITestExecutor>(pathToExtensions, loadOnlyWellKnownExtensions: true));
         }
 
+        [TestMethod]
+        public void GetTestExtensionsInformationShouldAddUWPCppAdatersIfTheyExists()
```

Comment:
Add the test for the other dll or include it in this one itself.

Reply:
resolved